src/common/JSHandle.ts

```diff
@@ -114,7 +114,7 @@ const applyOffsetsToQuad = (
  *
  * @public
  */
-export class JSHandle<HandleObjectType = unknown> {
+export class JSHandle<HandleObjectType = any> {
```

Comment:
I think unknown is much safer? Basically any allows us to do anything with the type (including what's not possible). Let's make it work with unknown.